1002-add-modalias-to-apple-bce.patch

```diff
@@ -11,13 +11,15 @@ diff --git a/drivers/staging/apple-bce/apple_bce.c b/drivers/staging/apple-bce/a
 index a6a656f..8cfbd3f 100644
 --- a/drivers/staging/apple-bce/apple_bce.c
 +++ b/drivers/staging/apple-bce/apple_bce.c
-@@ -432,5 +432,6 @@ MODULE_LICENSE("GPL");
+@@ -434,5 +434,6 @@ MODULE_LICENSE("GPL");
  MODULE_AUTHOR("MrARM");
  MODULE_DESCRIPTION("Apple BCE Driver");
  MODULE_VERSION("0.01");
 +MODULE_ALIAS("pci:v0000106Bd00001801sv*sd*bc*sc*i*");
  module_init(apple_bce_module_init);
- module_exit(apple_bce_module_exit);
- -- 
+-module_exit(apple_bce_module_exit);
```

Comment:
The patch adding and removing this line seems unnecessary.

Reply:
But if I remove this, git feels bad :(
So, I let it be that way

Reply:
When the patch is as below, it applies for me. Did you change the length of the before and after hunks when you edited the patch? (the ,5 and ,6 in the line with @@)
```diff
From 153b587ed53135eaf244144f6f8bdd5a0fe6b69e Mon Sep 17 00:00:00 2001
From: Redecorating <69827514+Redecorating@users.noreply.github.com>
Date: Fri, 24 Dec 2021 18:12:25 +1100
Subject: [PATCH 1/1] add modalias to apple-bce

---
 drivers/staging/apple-bce/apple_bce.c     |  1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/staging/apple-bce/apple_bce.c b/drivers/staging/apple-bce/apple_bce.c
index a6a656f..8cfbd3f 100644
--- a/drivers/staging/apple-bce/apple_bce.c
+++ b/drivers/staging/apple-bce/apple_bce.c
@@ -432,5 +432,6 @@ MODULE_LICENSE("GPL");
 MODULE_AUTHOR("MrARM");
 MODULE_DESCRIPTION("Apple BCE Driver");
 MODULE_VERSION("0.01");
+MODULE_ALIAS("pci:v0000106Bd00001801sv*sd*bc*sc*i*");
 module_init(apple_bce_module_init);
 module_exit(apple_bce_module_exit);
\ No newline at end of file
--
2.34.1
```

Reply:
check the latest commit